core-build/gulp-core-build/src/tasks/JestReporter.ts

```diff
@@ -6,15 +11,50 @@ const DEFAULT_REPORTER: any = require('jest-cli/build/reporters/default_reporter
  * rush does not get confused.
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: any): any }) {
+class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: Jest.GlobalConfig): any }) {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public constructor(globalConfig: any) {
+  private _globalConfig: Jest.GlobalConfig;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public constructor(globalConfig: Jest.GlobalConfig) {
     super(globalConfig);
+    this._globalConfig = globalConfig;
+
+    this._err = this._out;
   }
 
   public log(message: string): void {
     process.stdout.write(message + '\n');
   }
+
+  public onRunComplete(contexts: Set<Jest.Context>, results: Jest.AggregatedResult): void {
+    super.onRunComplete(contexts, results);
+
+    let outputPath: string | undefined = undefined;
+    let filename: string | undefined = undefined;
+
+    for (const reporter of (this._globalConfig.reporters as (string | ReporterConfig)[])) {
```

Comment:
The `(string | ReporterConfig)` type is nasty. Is that really what Jest provides?

Reply:
Yeeeeeep. They either expect 'string' to specify the reporter, or a tuple of [string, object] where object is the configuration of the reporter